pkgs/applications/networking/browsers/mullvad-browser/default.nix

```
@@ -0,0 +1,224 @@
+{ lib, stdenv
```

```suggestion
{ lib
, stdenv
```
This is only an artefact of a tree wife change